applications/tari_dan_wallet_daemon/src/main.rs

```diff
@@ -67,12 +67,7 @@ async fn main() -> Result<(), Box<dyn Error>> {
     let shutdown_signal = shutdown.to_signal();
 
     let store = SqliteWalletStore::try_open(cli.base_dir().join("data/wallet.sqlite"))?;
-    let mut migration_output = BufWriter::new(Vec::new());
-    store.run_migrations_with_output(&mut migration_output)?;
-    let migration_output = migration_output.into_inner()?;
-    if !migration_output.is_empty() {
-        log::info!(target: LOG_TARGET, "{}", String::from_utf8_lossy(&migration_output));
-    }
+    store.run_migrations()?;
```

Comment:
Don't we need to handle the migration outputs anymore ?

Reply:
I don't think having output is necessary really. As long as we get an error if it fails.